diff --git a/doomkeys.h b/fbdoom/doomkeys.h
index 2dfc431..a952b7a 100644
--- a/doomkeys.h
+++ b/doomkeys.h
@@ -19,6 +19,8 @@
 #ifndef __DOOMKEYS__
 #define __DOOMKEYS__
 
+#include <linux/input.h>
+
 //
 // DOOM keyboard definition.
 // This is the stuff configured by Setup.Exe.
@@ -33,7 +35,6 @@
 #define KEY_USE			0xa2
 #define KEY_FIRE		0xa3
 #define KEY_ESCAPE		27
-#define KEY_ENTER		13
 #define KEY_TAB			9
 #define KEY_F1			(0x80+0x3b)
 #define KEY_F2			(0x80+0x3c)

