diff --git a/doomkeys.h b/fbdoom/doomkeys.h
index 2dfc431..a952b7a 100644
--- a/doomkeys.h
+++ b/doomkeys.h
@@ -19,6 +19,8 @@
 #ifndef __DOOMKEYS__
 #define __DOOMKEYS__
 
+#include <linux/input.h>
+
 //
 // DOOM keyboard definition.
 // This is the stuff configured by Setup.Exe.
@@ -33,7 +35,6 @@
 #define KEY_USE			0xa2
 #define KEY_FIRE		0xa3
 #define KEY_ESCAPE		27
-#define KEY_ENTER		13
 #define KEY_TAB			9
 #define KEY_F1			(0x80+0x3b)
 #define KEY_F2			(0x80+0x3c)
diff --git a/i_sound.c b/fbdoom/i_sound.c
index 71947d2..f036e82 100644
--- a/i_sound.c
+++ b/i_sound.c
@@ -65,6 +65,7 @@ int snd_sfxdevice = SNDDEVICE_SB;
 extern void I_InitTimidityConfig(void);
 extern sound_module_t sound_sdl_module;
 extern sound_module_t sound_pcsound_module;
+extern sound_module_t sound_alsa_module;
 extern music_module_t music_sdl_module;
 extern music_module_t music_opl_module;
 
@@ -95,6 +96,7 @@ static sound_module_t *sound_modules[] =
     &sound_sdl_module,
     &sound_pcsound_module,
 #endif
+    &sound_alsa_module,
     NULL,
 };
 
